coordinator/src/trade/mod.rs

```diff
@@ -378,17 +412,19 @@ impl TradeExecutor {
                 margin_trader + collateral_reserve_trader + order_matching_fee,
                 dlc::FeeConfig::EvenSplit,
             ),
-            TraderRequiredLiquidity::None => (
-                margin_coordinator
-                    + collateral_reserve_coordinator.to_sat()
-                    + margin_trader
-                    + collateral_reserve_trader
-                    // If the trader doesn't bring their own UTXOs, including the order matching fee
-                    // is not strictly necessary, but it's simpler to do so.
-                    + order_matching_fee,
-                0,
-                dlc::FeeConfig::AllOffer,
-            ),
+            TraderRequiredLiquidity::None => {
+                (
+                    margin_coordinator
+                        + collateral_reserve_coordinator.to_sat()
+                        + margin_trader
+                        + collateral_reserve_trader
+                        // If the trader doesn't bring their own UTXOs, including the order matching fee
+                        // is not strictly necessary, but it's simpler to do so.
+                        + order_matching_fee,
+                    0,
+                    dlc::FeeConfig::AllOffer,
+                )
+            }
```

Comment:
🙃 I think nothing actually changed here.

Reply:
jap only the brakets have been added 🙈